charts/operator-wandb/values.yaml

```
@@ -1004,3 +1004,10 @@
     watchGlobally: false
     reloadOnCreate: true
     reloadOnDelete: true
+
+clickhouse:
+  install: false
+  shards: 1
+  auth:
+    existingSecret: '{{ include "wandb.clickhouse.passwordSecret" . }}'
+    existingSecretKey: '{{ include "wandb.clickhouse.passwordSecret.passwordKey" . }}'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)